packages/registry/test/registry/registry.spec.tsx

```diff
@@ -35,6 +35,10 @@ describe('Registry', () => {
     it('throws if a component name is not valid', () => {
       expect(() => Registry.getComponentMetadata(InvalidNameComp)).to.throw();
     });
+
+    it('returns default value equals to true for a11yCompliant', () => {
```

Comment:
This test is in the wrong place. It should be in the `component-metadata.spec.tsx` file.

But, although testing is good (👍 ) I actually don't think we need a test for this